src/reducers/index.js

```diff
@@ -1,16 +1,22 @@
-import { INCREMENT, DECREMENT } from '../actions';
+import { INCREMENT, DECREMENT, ODDINCREMENT } from '../actions';
 
 // Our reducer that handles our two action cases:
 // increment and decrement. It receives the state
 // of our redux store, along with an action created
 // by our action creator. What does the reducer
 // need to do with the count in each case?
+
 export default (count = 0, action) => {
   switch (action.type) {
     case INCREMENT:
-    // Fill in the body of this case
+    return count = action.payload + count
```

Comment:
`++count` and `--count` are great shorthand ways to increment or decrement a variable. Take a look at the difference however between `++count` vs. `count++`. Don't overthink, payload wasn't necessary.